src/Cli/dotnet/Installer/Windows/InstallRequestMessage.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#nullable disable
+
+using System.Text;
+using Microsoft.DotNet.Workloads.Workload.Install.InstallRecord;
+using Newtonsoft.Json;
+
+namespace Microsoft.DotNet.Installer.Windows
+{
+    /// <summary>
+    /// Defines the IPC message structure used to send commands from the unelevated client to the elevated server instance of
+    /// the installer
+    /// </summary>
+    internal class InstallRequestMessage : InstallMessageBase
+    {
+        /// <summary>
+        /// The dependent value to modify.
```

Comment:
That's used for reference counting. Typically, it would be ```Microsoft.NET.Sdk,<feature-band>,<SDK host architecture>```.

![image](https://user-images.githubusercontent.com/7409981/124842417-57476900-df44-11eb-8efc-2d7d6e1b946d.png)

Reply:
Ah, dependent in the registry? Can we include that in the comment?

Reply:
I've pushed an update for the comments